.github/assets/hive/run_simulator.sh

```
@@ -1,14 +1,32 @@
 #!/usr/bin/env bash
-# Copied from reth v1.8.1
+# Synced with reth v2.4.0
 # set -x
 
 cd hivetests/
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)